### PR TITLE
docs(configuration): document resolve.roots

### LINKED ISSUE #3826: Document webpack change: add roots option
*A pull request by @vankop was merged and maintainers requested a documentation change.*

See pull request: https://github.com/webpack/webpack/pull/11143

---

**What kind of change does this PR introduce?**

feature

**Did you add tests for your changes?**

updated snapshots

**Does this PR introduce a breaking change?**

no

**What needs to be documented once your changes are merged?**

webpack supports `options.resolve.roots` option. Defaults to `options.context`. 

If request is an server-relative URL (starting with `/`)  `webpack` will try to resolve `path.join(root[i], request)` after trying to resolve the path as absolute path on non-windows systems

### PR DESCRIPTION
**Pull request by chenxsan**
closes #3826

### CODE REVIEW

File: src/content/configuration/resolve.md [hunk 1]
```diff
@@ -470,6 +470,24 @@ module.exports = {
 };
 ```
 
+### `resolve.roots`
+
+`[string]`
+
+A list of directories where requests of server-relative URLs (starting with '/') are resolved. On non-windows system these requests are resolved as absolute path first.
```
Comment (EugeneHlushko) [inline src/content/configuration/resolve.md:477]: Can we say `systems`? Its more common these days
```suggestion
A list of directories where requests of server-relative URLs (starting with '/') are resolved. On non-windows systems these requests are resolved as absolute path first.
```

**Comment by EugeneHlushko** [comment]
From issue:
"options.resolve.roots option Defaults to options.context."

File: src/content/configuration/resolve.md [hunk 2]
```diff
@@ -470,6 +470,24 @@ module.exports = {
 };
 ```
 
+### `resolve.roots`
+
+`[string]`
+
+A list of directories where requests of server-relative URLs (starting with '/') are resolved, defaults to `[options.context]`. On non-windows systems these requests are resolved as absolute path first.
```
Comment (montogeek) [inline src/content/configuration/resolve.md:477]:
```suggestion
A list of directories where requests of server-relative URLs (starting with '/') are resolved, defaults to `[options.context]`. On non-Windows systems these requests are resolved as an absolute path first.
```

File: src/content/configuration/resolve.md [hunk 3]
```diff
@@ -470,6 +470,24 @@ module.exports = {
 };
 ```
 
+### `resolve.roots`
+
+`[string]`
+
+A list of directories where requests of server-relative URLs (starting with '/') are resolved, defaults to `[options.context]`. On non-Windows systems these requests are resolved as an absolute path first.
```
Comment (EugeneHlushko) [inline src/content/configuration/resolve.md:477]:
```suggestion
A list of directories where requests of server-relative URLs (starting with '/') are resolved, defaults to [`context` configuration option](/configuration/entry-context/#context). On non-Windows systems these requests are resolved as an absolute path first.
```

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**